### PR TITLE
Typography

### PR DESCRIPTION
**Pull request by smialko**
I added some typography styles based on Figma designs for future use.

### CODE REVIEW

File: app/javascript/react/assets/styles/Typography/Typography.style.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
+import styled, { css } from "styled-components";
+
+const H1 = styled.h1`
+  font-family: "Roboto";
```
Comment (franiakat) [inline app/javascript/react/assets/styles/Typography/Typography.style.tsx:4]: I think there is no other font used in the calendar feature, why don't we just set it once in `global-styles.ts`?

File: app/javascript/react/assets/styles/Typography/Typography.style.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  font-weight: 600;
+  font-size: 28px;
+  font-style: normal;
+  line-height: 1.2;
+  color: #3e4449;
```
Comment (franiakat) [inline app/javascript/react/assets/styles/Typography/Typography.style.tsx:9]: Please add this color to variables, also we can set the font color globally, as most often that would be true, can be then removed from all the elements.

File: app/javascript/react/assets/styles/Typography/Typography.style.tsx [hunk 3]
```diff
@@ -0,0 +1,107 @@
+import styled, { css } from "styled-components";
+
+const H1 = styled.h1`
+  font-family: "Roboto";
+  font-weight: 600;
+  font-size: 28px;
+  font-style: normal;
```
Comment (franiakat) [inline app/javascript/react/assets/styles/Typography/Typography.style.tsx:7]: It's a default, don't have to specify that, please remove.

File: app/javascript/react/assets/styles/Typography/Typography.style.tsx [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+  line-height: 1.2;
```
Comment (franiakat) [inline app/javascript/react/assets/styles/Typography/Typography.style.tsx:8]: Could be globally set as well I think?

File: app/javascript/react/assets/styles/Typography/Typography.style.tsx [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+`;
+
+const H3bold = styled.h3`
+  font-family: "Roboto";
+  font-weight: 600;
+  font-size: 18px;
+  font-style: normal;
+  line-height: 1.2;
+  color: #3e4449;
+`;
```
Comment (franiakat) [inline app/javascript/react/assets/styles/Typography/Typography.style.tsx:38]: Here we can have just one component with a prop:
```suggestion
const H3 = styled.h3<{bold?: boolean}>`
  font-weight: ${(p) => p.bold ? '600' : '400'};
  font-size: 18px;
`;
```

I've written it here, but most probably should work.

File: app/javascript/react/assets/styles/Typography/Typography.style.tsx [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+const H6uppercase = styled.h6`
+  font-family: "Roboto";
+  font-weight: 400;
+  font-size: 10px;
+  font-style: normal;
+  line-height: 1.2;
+  letter-spacing: 0.1px;
+  text-transform: uppercase;
+  color: #3e4449;
+`;
```
Comment (franiakat) [inline app/javascript/react/assets/styles/Typography/Typography.style.tsx:105]: We can treat this uppercase feature as a prop as well, similar to bold.

**Review by franiakat — CHANGES_REQUESTED** [review]
⭐ Happy to be doing reviews again!

Please see the suggestions. :)

**Review by jankrzempek — COMMENTED** [review]
It was a great opportunity to learn, thanks. Nothing more to add 😄

**Review by franiakat — APPROVED** [review]
Good to go! 🎖️ 
Please use the option of squash and merge.
sqash

**Pull request merged**